webknossos-datastore/app/com/scalableminds/webknossos/datastore/dataformats/zarr/ZarrCoordinatesParser.scala

```diff
@@ -1,15 +1,40 @@
 package com.scalableminds.webknossos.datastore.dataformats.zarr
 
+import com.scalableminds.util.tools.Fox
+import com.scalableminds.util.tools.Fox.{bool2Fox, option2Fox}
+import com.scalableminds.webknossos.datastore.models.AdditionalCoordinate
+import play.api.http.Status.NOT_FOUND
+import play.api.i18n.{Messages, MessagesProvider}
+
+import scala.concurrent.ExecutionContext
+
 object ZarrCoordinatesParser {
-  def parseDotCoordinates(
-      cxyz: String,
-  ): Option[(Int, Int, Int, Int)] = {
-    val singleRx = "\\s*([0-9]+).([0-9]+).([0-9]+).([0-9]+)\\s*".r
-
-    cxyz match {
-      case singleRx(c, x, y, z) =>
-        Some(Integer.parseInt(c), Integer.parseInt(x), Integer.parseInt(y), Integer.parseInt(z))
-      case _ => None
-    }
```

Comment:
This was removed as it was fully replaced by `parseNDimensionalDotCoordinates`. This was already the case in the first version of this pr